yesod-test/Yesod/Test.hs

```diff
@@ -93,6 +95,7 @@ module Yesod.Test
     -- * Grab information
     , getTestYesod
     , getResponse
+    , getRequestCookies
```

Comment:
I don't know the best spot for this function, either in the export list or in the source code.